juniper/src/types/name.rs

```diff
@@ -6,7 +6,7 @@ use std::{
 };
 
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct Name(String);
```

Comment:
@tusharmath I don't like this change, because it breaks encapsulation for the `Name` type here and allow freely bypass its invariants. It's far better here to provide a meaningful constructor-function, which checks the invariants.